### ISSUE TITLE
Typos in the helloworld example

### ISSUE THREAD

**Issue by gabrik** [eclipse-fog05/website#20]
There are few typos in the "Hello World!" example.

 - Missing import of time
 - Missing line taht defined fdu_id `fdu_id = fduD.get_uuid()`
 - Missing variable in the print for instance id `print ('Instance ID : {}'.format(inst_info.get_uuid()))`

**Comment by gabrik** [comment]
This one works.

```
import json
import time
from fog05 import FIMAPI
from fog05_sdk.interfaces.FDU import FDU

def read_file(filepath):
	with open(filepath, 'r') as f:
		data = f.read()
	return data

api = FIMAPI()
desc = json.loads(read_file('/home/ubuntu/fdu_helloworld.json'))
fdu_descriptor =  FDU(desc)
fduD = api.fdu.onboard(fdu_descriptor)
fdu_id = fduD.get_uuid()
print ( 'fdu_id: {}'.format(fduD.get_uuid()))
time.sleep(2)
inst_info=api.fdu.define(fdu_id)

api.fdu.configure(inst_info.get_uuid())
time.sleep(2)
api.fdu.start(inst_info.get_uuid(), "MYENV=Hello World!")
print('Instance ID: {}'.format(inst_info.get_uuid()))
time.sleep(1)
log = api.fdu.log(inst_info.get_uuid())
print('FDU Output:\n{}\n'.format(log))

input('Press enter to terminate')

api.fdu.terminate(inst_info.get_uuid())

api.close()
exit(0)

```

**Comment by gabrik** [comment]
fixed by #21
